src/main/webapp/filterScript.js

```diff
@@ -0,0 +1,43 @@
+const UNCLICKED = 'unclicked-filter';
+const CLICKED = 'clicked-filter';
+const CLICKED_ARRAY_URL = 'clicked-array=';
```

Comment:
The name of this constant should include the information that this is the name of a parameter key. So `CLICKED_ARRAY_URL_PARAM` or `CLICKED_ARRAY_KEY` or something would be more clear. Also, it's more common to keep the `=` out of the constant value in situations like this and just apply it when used, like `CLICKED_ARRAY_KEY + '=' value`. This design gives you more flexibility, e.g. you could use the same constant when reading data out of a URL (not relevant here, but does come up in some situations) or using a more structured URL creation method.